src/routes/landing/containers/LandingContainer.js

```diff
@@ -24,7 +25,28 @@ const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>, ownProps) => {
       type: 'FETCH_CITIES',
       params: {}
     }),
-    navigateToCategory: createNavigateToCategory('Dashboard', dispatch, ownProps.navigation)
+    navigateToDashboard: (cityCode: string, language: string) => {
```

Comment:
Why did you have to change this? Is it because of the key and the problem I also have in #57 ?

Reply:
Can't be maybe just create a separate helper function `createNavigateToDashboard`? Mustn't belong to this task though.

Reply:
Yes, it was because of the key problem and
Yes, we could create a separate helper function, but we need a specific one to go *from* Landing *to* Dashboard (because we need to address the `switch`-router here as well. So this would be the only occurence here...

Reply:
So I think this should be fine as is.